Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py

```diff
@@ -187,7 +203,8 @@ def http_request(self, method: str, url_suffix: str, json_data=None, params=None
         """
         header = {
             "Authorization": "Bearer {token}".format(token=self.api_key),
-            "Content-Type": f'{CONTENT_TYPE_JSON}{";charset=utf-8"}'
+            "Content-Type": f'{CONTENT_TYPE_JSON}{";charset=utf-8"}',
+            "User-Agent": f'{USER_AGENT}'
```

Comment:
```suggestion
            "Content-Type": f'{CONTENT_TYPE_JSON};charset=utf-8',
            "User-Agent": USER_AGENT,
```

Reply:
Thanks for the suggestion. Change incorporated.